community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -529,8 +529,24 @@ public String providerName()
         }
     }
 
-    @Description( "Index provider to use when creating new indexes." )
-    public static final Setting<String> default_schema_provider =
+    @Description( "Index provider to use for newly created schema indexes." +
+            "An index provider may store different value types in separate physical indexes." +
+            "lucene-1.0: Store spatial and temporal value types in native indexes, remaining value types in a Lucene index." +
+            "lucene+native-1.0: Store numbers in a native index and remaining value types like lucene-1.0." +
+            "This improves read and write performance for non-composite indexed numbers." +
+            "lucene+native-2.0: Store strings in a native index and remaining value types like lucene+native-1.0." +
+            "This improves write performance for non-composite indexed strings." +
+            "This version of the native string index has a value limit of 4047B, such that byte-representation" +
+            "of a string to index cannot be larger than that limit, or the transaction trying to index such a value will fail." +
+            "This version of the native string index also has reduced performance for CONTAINS and ENDS WITH queries," +
+            "due to resorting to index scan+filter internally." +
+
```

Comment:
Random blank line.

Reply:
Deliberate, but perhaps it's against how these descriptions are commonly written?

Reply:
It looks like you intended to have a blank line in the description. It's in the code, but not in the string, which follows from my comment below.

Reply:
Huh, how silly of me